community/pmount/depends

```diff
@@ -0,0 +1,3 @@
+autoconf make
+util-linux
+
```

Comment:
`autoconf` should instead be `automake` as `autoreconf` will use `aclocal`. 
Remove the blank line at the end of this.